### ISSUE TITLE
A seed for reproducibility?

### ISSUE THREAD

**Issue by yug125lk** [leondgarse/Keras_insightface#136]
Thank you again. I have another question about the seed. In your code, did you set a seed to reproduce the results? When I trained the model twice, I got different results.

**Comment by leondgarse** [comment]
Seldom do I set a seed, and most times the results are near. Did your results differ a lot?

**Comment by yug125lk** [comment]
I trained twice ResNet34 on the CASIA-WebFace dataset, using Swish activation. On LFW gave me 99.35 and 99.21. I thought it might be an issue related to the seed!!

**Comment by leondgarse** [comment]
Yes, that difference may be due to the seed. May try to save a randomly initialized model at first and use that for later other training tests.
